data/json/items/armor/boots.json

```diff
@@ -712,6 +712,37 @@
     "proportional": { "weight": 0.75, "volume": 0.75, "price": 0.5 },
     "extend": { "flags": [ "UNDERSIZE" ] }
   },
+  {
+    "id": "sabaton_metal_sheets",
+    "type": "ARMOR",
+    "name": { "str": "pair of sheet metal sabatons", "str_pl": "pairs of sheet metal sabatons" },
+    "description": "Some sheets of metal hammered to fit over the top of your foot and around your ankles, meant to be worn over boots.  They make walking a fair bit more awkward, but they'll keep your toes from being stomped on.",
+    "//": "The weight assumes two small sheets of about 20cm x 10cm and two small sheets of about 30cm x 10 cm.",
+    "weight": "1180 g",
+    "volume": "750 ml",
+    "price": 400,
+    "price_postapoc": 200,
+    "to_hit": -2,
+    "bashing": 7,
+    "symbol": "[",
+    "looks_like": "lc_sabaton",
+    "material": [ "lc_steel" ],
+    "repairs_with": [ "steel" ],
```

Comment:
lc_steel already repairs with chunks of low carbon steel, why is this being overwritten 🤔

~~Also I expected Sabatons to be more musical~~

Reply:
It's a lot more difficult and more effort to make low carbon steel chunks than just using regular steel chunks, and the armor is meant to be crude enough that welding any steel into it should be fine. Especially since none of the sheet metal armor is "STURDY" by design.

I originally wanted to make them repairable with small metal sheets instead but those aren't a valid repair item.

Basically they should be relatively easy to repair but you need an entire forge and charcoal production to make LC lumps, which you then have to disassemble into LC chunks to repair LC steel with.